rest-api/tools/import_organizations.sh

```diff
@@ -35,7 +34,12 @@ function get_geocode_key {
 }
 
 CREDS_ACCOUNT="${ACCOUNT}"
+if [ -z "${ACCOUNT}" ]
+then
+echo "Not Geocoding addresses without --account"
```

Comment:
You need to change this logic:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/tools/import_organizations.py#L210

So if api_key isn't set we don't try to do any geocoding.

Reply:
in progress, trouble passing the flag into .py

Reply:
When dealing with boolean flags like this, the flag you pass in is either '--geocode_flag' for true or '--nogeocode_flag' (or simpler, the absence of --geocode_flag) for false. The way you have it now, you're passing in literally --true or --false, which doesn't make sense.